communication/slack-guidelines.md

```diff
@@ -1,90 +1,232 @@
+sales pitch. Admins will take a screenshot of such behavior in order to document
+the situation. The community takes such violations extremely seriously, and the
+admins are empowered to handle it swiftly.
+
+
+### Sending Messages to the Channel
+
+`@all`, `@here` and `@channel` should be used rarely. Members will receive
+notifications from these commands. Remember Kubernetes is a global project -
+please be kind.
+
+
+### Slack Requests
+
+Admins are tasked with processing requests for channels and other things such as
+bots, tokens or webhook.
+
+
+#### Channel Requests
+
+Channel requests should be reviewed for their relation and relevance to the
+Kubernetes community. Typically channels should be dedicated to SIGs, WGs, UGs,
+sub-projects, community topics, and other things related to Kubernetes programs
+and projects.
+
+For Kubernetes project centric requests, validate them against the [sig-list],
+or request a link to a related issue/PR, or mailing list discussion for the
+requested Channel.
+
+Small external projects are encouraged to use the channel of the SIG, WG, or UG
+most relevant to them. Other things such as programming language specific
+channels are discouraged and should in turn be steered to `#kubernetes-client`
+or communication avenues commonly used by their specific language.
+
+In general, use your best judgment.
+
+Once two admins have agreed to sponsor the channel, an admin should assign the
+issue to themselves, and create the channel. A message should then be pinned
+with the below text:
+```
+This channel abides to the Kubernetes Code of Conduct - http://git.k8s.io/community/code-of-conduct.md
+Contact conduct@kubernetes.io or an admin in the #slack-admins channel if there is a problem.
+```
+
+
+#### Bot, Token, or Webhook Requests
+
+Requests should first be evaluated for their relevance to the project. Typically
+approved requests are related to: GitHub, the CNCF, or other tools/platforms
+used to aid in the management of Slack. Requests outside of this scope should be
+heavily scrutinized and reviewed for **ANY** potential security, privacy, or
+usability concerns.
+
+It is best to err on the side of not allowing a Bot, Token or Webhook request
+than allowing one.
+
+If consensus among the admins can be reached regarding the request; an admin
+should assign the issue to themselves, and reach out to the request contact
+regarding next steps.
+
+
+### Inactivating Accounts
+
+For the reasons listed below, admins may inactivate individual Slack accounts.
+Due to Slack’s framework, it does not allow for an account to be banned or
+suspended in the traditional sense, merely inactivated. See [Slack’s policy on
+inactivated accounts] for more information.
+
+- Spreading spam content in DMs and/or channels.
+- Not adhering to the code of conduct set forth in DMs and/or channels.
+- Overtly selling products, related or unrelated to Kubernetes.
+
+---
+
+## Requesting a Channel
```

Comment:
How does it combine with the "#### Channel Requests" subsection above?

Reply:
One is a process for users to follow, the other is a process for admins to follow to process their request. I originally had it combined, but it seemed appropriate to break it apart. Would changing the other heading to be something like "Processing Slack Requests" Make it a clearer distinction from requesting one?

Reply:
Other thing I can do to make it clearer would be to move the 'request channel/request bot portions beneath the general communication guidelines which is more user focused. That way the moderation section doesn't break the overall flow of doc.

Reply:
@mrbobbytables +1

Reply:
Relocated and made it a bit clearer.